src/redux/reducers/todo.reducer.ts

```diff
@@ -0,0 +1,44 @@
+/* TODO: refactor and split types separately */
```

Comment:
Why is it needed in this task?

Reply:
Removed irrelevant (for this task) TODO Reducer and reinitialized store with an empty object.